docs/how_to/nginx_https.md

```diff
@@ -0,0 +1,49 @@
+# How to use Nginx to serve Livebook over HTTPS
+
+This guide shows you how to use Nginx to serve Livebook over HTTPS.
+
+## Prerequisites
+
+- Nginx installed
+- Livebook installed
+- Both Livebook and Nginx running on the same machine or within the same network
+- SSL certificate and key files
+
+## Nginx configuration
+
+Use the following Nginx config file as a starting point:
+
+```nginx
+http {
+    server {
+        listen 443 ssl;
+        server_name your_domain_or_ip;  # e.g., livebook.example.com
```

Comment:
Does it not _have_ to be a domain registered for the certificate to be used correctly?